draft-ietf-jsonpath-base.md

```diff
@@ -1,7 +1,7 @@
 ---
 v: 3
 
-docname: draft-ietf-jsonpath-base-latest
+docname: draft-ietf-jsonpath-base-14
```

Comment:
This is probably not really part of the PR?

Reply:
Whoops! Reverted.